### ISSUE TITLE
FAQ-BOT bad return to WhatsApp server

### ISSUE THREAD

**Issue by traderpedroso** [hanzala-sohrab/baileys-whatsapp-server#1]
TypeError: Cannot read properties of undefined (reading 'text')

**Comment by hanzala-sohrab** [comment]
@traderpedroso this server needs to run in conjunction with this [Rasa server](https://github.com/hanzala-sohrab/faq-bot).
This server handles the WhatsApp APIs and the Rasa server handles the NLU part

**Comment by traderpedroso** [comment]
Yes its runing ok server and Client but some mensagens break WhatsApp api and return The error TypeError: Cannot read properties of undefined (reading 'text')'
